.babelrc

```diff
@@ -1,7 +1,9 @@
 {
   "presets": ["react", "es2015"],
   "plugins": [
-    "transform-object-assign"
+    "transform-object-assign",
+    "transform-proto-to-assign",
```

Comment:
And this one transforms setting `__proto__` to regular property assignment. Older runtimes are not aware of `__proto__`.